consumer/push_consumer.go

```diff
@@ -175,6 +191,7 @@ func (pc *pushConsumer) Start() error {
 
 func (pc *pushConsumer) Shutdown() error {
 	pc.lockTicker.Stop()
+	close(pc.done)
```

Comment:
it is better that add a `sync.Once` to wrap the block to avoiding double call

Reply:
@wenfengwang done

Reply:
@wenfengwang done